### PR TITLE
BROOKLYN-316: fix use of karaf jetty LoginService

### PR DESCRIPTION
**Pull request by aledsage**

### CODE REVIEW

File: karaf/jetty-config/src/main/resources/jetty.xml [hunk 1]
```diff
@@ -56,6 +56,9 @@
                         <Item>org.apache.brooklyn.rest.security.jaas.BrooklynLoginModule$RolePrincipal</Item>
                     </Array>
                 </Set>
+                <Set name="identityService">
+                    <New class="org.eclipse.jetty.security.DefaultIdentityService"/>
+                </Set>
```
Comment (neykov) [inline karaf/jetty-config/src/main/resources/jetty.xml]: That's the default if none is specified (see [code](https://github.com/eclipse/jetty.project/blob/jetty-9.3.x/jetty-jaas/src/main/java/org/eclipse/jetty/jaas/JAASLoginService.java#L178)).

Comment (m4rkmckenna) [inline karaf/jetty-config/src/main/resources/jetty.xml]: @neykov But if two threads are invoking that code an `java.lang.IllegalStateException` is thrown

**Comment by neykov** [comment]
Reading the the jira issue, the change makes much more sense now.
Shouldn't you add the package `org.eclipse.jetty.security` to the imports instead?

**Comment by m4rkmckenna** [comment]
LGTM / tested

**Comment by aledsage** [comment]
@neykov you said: "add the package org.eclipse.jetty.security to the imports instead." Do you mean in karaf/jetty-config/pom.xml ? I've added that now.

Merging now.

**Pull request merged**